src/main/java/com/accolite/pru/health/AuthApp/security/JwtTokenProvider.java

```diff
@@ -40,11 +45,17 @@ public JwtTokenProvider(@Value("${app.jwt.secret}") String jwtSecret, @Value("${
      */
     public String generateToken(CustomUserDetails customUserDetails) {
         Instant expiryDate = Instant.now().plusMillis(jwtExpirationInMs);
+        String authorities = customUserDetails
+                .getAuthorities()
+                .stream()
+                .map(GrantedAuthority::getAuthority)
+                .collect(Collectors.joining(","));
```

Comment:
Can be extracted into a new method to make it cleaner. `generateToken` shouldn't own the logic to fetch `authorities`. It should delegate